### ISSUE TITLE
nx@6 workspace  diretory structure proposal 

### ISSUE THREAD

**Issue by xmlking** [nrwl/nx#456]
Can we have three top level directories instead of just apps/libs ?
this way we can organize code into internal reusable modules (private) in `modules` and publishable (public)   module and libs in `libs`
> the choice for naming 3rd directory  `libs` or `packages` 

```
apps
  -app1
  -app2
  -mobileApp1
modules   - contain shared/future ng-modules - private
  -shared-module
  -future-module1
  -future-module2
libs  - contain sharable/publishable ng-modules and pure JS libs - public
  -upload-module
  -animations
  -my-lodash
```

**Comment by kylecordes** [comment]
In work I did prior to Nx existing, manually performing similar gymnastics to what Nx does, I also divided reusable chunks into three categories as described above. It worked out very nicely, and I recommend it.

(However, the Nx tooling is approximately 50 times more convenient than what I was doing manually, so I switched to Nx.)

**Comment by xmlking** [comment]
@kylecordes Agree. #nx can still add value on top of  `angular-cli@6`,  like enhanced generators, `tslint` rules, tags , `dep-graph`, `affected`,  `format` etc.

**Comment by xmlking** [comment]
I think nx@6 ‘s   apps + libs directory structure with —publishable  flag satisfy my needs.  I like simplicity of old non-publishable lib modules. Why generate tslint, tsconfig... files for each non-publishable lib modules?
